helmfile/charts/networkpolicy-service/templates/calico-accountant/allow-calico-accountant.yaml

```diff
@@ -0,0 +1,110 @@
+{{ if .Values.calicoAccountant.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-calico-accountant
+spec:
+  podSelector:
+    matchLabels:
+      app: calico-accountant
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
```

Comment:
I'm confused by this whole file, I did not think that calico accountant needed to talk to this many things.

To help me understand, could you list which things it talks to and why (both ingress and egress)?